ignite-worker/src/main/java/org/opennms/poc/ignite/worker/workflows/WorkflowLifecycleManager.java

```diff
@@ -0,0 +1,10 @@
+package org.opennms.poc.ignite.worker.workflows;
+
+import org.opennms.poc.ignite.grpc.whiteboard.api.MessageListener;
+import org.opennms.poc.ignite.model.workflows.Workflows;
+
+/**
+ * Watcher for Workflow definitions responsible to manage worker lifecycles to ensure all Workflows are processed.
+ */
+public interface WorkflowLifecycleManager extends MessageListener<Workflows> {
```

Comment:
Is this just a marker interface?

Reply:
It started as something more useful, but then `MessageListener` came along.  I considered to remove it, but the IMPL class is still called IMPL and is a sub-folder, so I left it for now.

Would be safe to simply remove it.  I think the idea of `WorkflowLifecycleManager` directly being a `MessageListener` is a little unclear, but adding another class to the mix feels like overkill too.